node/src/ops/Reduce.h

```diff
@@ -0,0 +1,27 @@
+#ifndef NODE_OPS_REDUCE_H_
+#define NODE_OPS_REDUCE_H_
+
+#include <napi.h>
+#include <webnn/webnn_cpp.h>
+
+namespace node { namespace op {
+
+    enum ReduceOpType {
```

Comment:
Simplify it as `ReduceType`

Reply:
Yes, OV hasn't supported now.

Reply:
How about `DML` and `NNAPI`, should we report to WebNN Spec?